### PR TITLE
data card germany

### PR DESCRIPTION
**Pull request by krsnapaudel**
* Data card
* Data preparation notebook

### CODE REVIEW

File: data_preparation/crop_statistics_DEU/README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+potat_tot | Potatoes
+wrape | Winter rape
+rye | Rye
+sugarbeet | Sugarbeet
+triticale | Triticale
+ww | Winter wheat
+
+**Temporal coverage**: 1979 - 2021
+
+**Temporal resolution**: 397 districts
```
Comment (ellaampy) [inline data_preparation/crop_statistics_DEU/README.md:67]: please revise here

**Review by ellaampy — CHANGES_REQUESTED** [review]
please check spatial and temporal resolution

File: data_preparation/crop_statistics_DEU/README.md (hunk 1 above)
Comment (krsnapaudel) [reply data_preparation/crop_statistics_DEU/README.md:67]: @CarlaLimone : Could you please address Stella's comments.

**Review by ellaampy — CHANGES_REQUESTED** [review]
@CarlaLimone please push your changes. It has not reflected yet.

**Review by ellaampy — APPROVED** [review]

**Pull request merged**